deploy/crds/kieapp.crd.yaml

```diff
@@ -637,6 +643,9 @@ spec:
                       type: object
                       description: Configuration of the RHPAM smart router
                       properties:
+                        context:
```

Comment:
maybe `imageContext` is better here? to give user a better understanding of its purpose?